test/e2e/framework/pod/wait.go

```diff
@@ -196,12 +188,11 @@ func WaitForPodsRunningReady(ctx context.Context, c clientset.Interface, ns stri
 			var buffer strings.Builder
 			buffer.WriteString(fmt.Sprintf("Expected all pods (need at least %d) in namespace %q to be running and ready (except for %d).\n", minPods, ns, allowedNotReadyPods))
```

Comment:
Looking through Git history, it seems like this file used to have quite a bit of logging, but not so much anymore. I considered adding additional logging here for failed Pods, but decided that only have this returned error string is most consistent with the style I see elsewhere.